bundles/CoreBundle/Migrations/Version20201222101114.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20201222101114 extends AbstractPimcoreMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->addSql('RENAME TABLE translations_website TO translations_messages;');
```

Comment:
I guess this is a bc-break, we should do the renaming in v10 and keep the current table name for now. 
We did that also here #7263

Reply:
We can think of using a view as kind of an alias, I guess that would work as well. 
Then in v10 we just drop the BC view.

Reply:
Added BC view.